### DISCUSSION TITLE
xiaomi apks

### DISCUSSION THREAD

**Discussion by xBZZZZ** [Universal-Debloater-Alliance/universal-android-debloater-next-generation#42] [APK sharing]
sharing `com.wingtech.standard` apk because description:https://github.com/Universal-Debloater-Alliance/universal-android-debloater/blob/420b0cc336ea9e44d56768c825598944d23af4d8/resources/assets/uad_lists.json#L17394
phone info:<table>
<tr><td>Device model</td><td>Redmi 6</td></tr>
<tr><td>MIUI version</td><td>MIUI Global 11.0.4<br>Stable<br>11.0.4.0(PCGMIXM)</td></tr>
<tr><td>Android version</td><td>9 PPR1.180610.011</td></tr>
<tr><td>Android security patch level</td><td>2020-05-01</td></tr>
</table>

gzipped apk download: https://github.com/Universal-Debloater-Alliance/universal-android-debloater/files/13321308/WTStandardTest.apk.gz

**Comment by xBZZZZ** [comment]
https://github.com/Universal-Debloater-Alliance/universal-android-debloater/blob/420b0cc336ea9e44d56768c825598944d23af4d8/resources/assets/uad_lists.json#L15934-L15936
here is `com.mediatek.gpslocationupdate` apk (same phone) gzipped download: https://github.com/Universal-Debloater-Alliance/universal-android-debloater/files/13340140/GPSLocationUpdate.apk.gz

**Comment by AnonymousWP** [comment]
Is this an improvement of the description or what exactly? Anyway, you should have created an issue instead, unless I'm missing your point here.

**Reply by xBZZZZ** [reply]
> Is this an improvement of the description

No, my posts are not asking to change descriptions.
> or what exactly?

There are parts of some descriptions like `Can someone share the apk just to be 100% sure?` and `Can someone share the apk? (from a Xiaomi/Huawei phone)`.

I can't find apks on https://www.google.com/, so I uploaded them to github.
> you should have created an issue instead

why? this is not bug report

**Reply by AnonymousWP** [reply]
Could you then explain me the purpose of this post?😅 You simply wanted to upload those APKs for research purposes?

**Reply by xBZZZZ** [reply]
> You simply wanted to upload those APKs for research purposes?

yes

**Comment by AnonymousWP** [comment]
@Blitz517 Maybe interesting for you to look at?
